### ISSUE TITLE
Setup README.md Layout

### ISSUE THREAD

**Issue by simba328** [KerasKorea/KerasObjectDetector#2]
I've just set up the layout of README.md. Please leave a comment if you have something to edit or add.
(The logo is a temporary image.)

# Keras Object Detection API (YOLK)

_You Only Look Keras :fried_egg:_

![Object](https://img.shields.io/badge/Object-Detector-Yellow.svg)
![Language](https://img.shields.io/badge/Language-Python-blue.svg)
[![DeepLearning](https://img.shields.io/badge/DeepLearning-Keras-red.svg)](https://keras.io)
[![KerasKorea](https://img.shields.io/badge/Community-KerasKorea-purple.svg)](https://www.facebook.com/groups/KerasKorea/)
[![KerasKorea](https://img.shields.io/badge/2019-Contributhon-green.svg)](https://www.kosshackathon.kr/)

<p align="center">
  <img width="509" height="276" src="https://user-images.githubusercontent.com/23257678/65056804-81fdb180-d9ac-11e9-931b-d027649c67cc.png" alt="">
</p>

As a 2019 Open Source Countibuthon Project, we create Kears Object Detection API.  
The purpose of Keras Object Detection API is to make it easy and fast so that everyone can create their own Object Detection model without difficulty.  
Detailed instructions for use will be updated at a later. You can look forward to it. 🤓

## Contents

* [Installation](#Installation)
* [Testing](#Testing)
* [Usage](#Usage)
* [Setup environment](#Setup-environment)
* [Release information](#Release-information)
* [Contributors](#Contributors)

## Setup environment

||version|
|---|---|
|python|3.6|
|tensorflow|1.14.0|
|keras|2.2.4|

**Comment by MijeongJeon** [comment]
Hi, simba328
YOLK logo is sooo cute and contents list is well constructed.
Could you make a `readme` branch and commit-push-pull request with it!?
Thanks!

**Comment by mike2ox** [comment]
Oh! Thank you, @simba328 
And, I think it's good to put a picture of a team member like this [README](https://github.com/jojoldu/junior-recruit-scheduler). :)

**Comment by mike2ox** [comment]
@simba328 @MijeongJeon 
I uploaded README.md to 'readme' branch. Plese check the contents and leave comments if there's anything missing or wrong. Thank you 😃

**Comment by MijeongJeon** [comment]
@mike2ox 
Thanks for well-arranged readme! 
Could you please add some images which are tested with object detection model?
